src/views/Wallet/views/AccountReceive/index.js

```diff
@@ -5,6 +5,7 @@ import { H2 } from 'components/Heading';
 import Button from 'components/Button';
```

Comment:
- <eye> tooltip has strange padding and its not properly aligned
- there are some fonts error in console
- i'm not sure if "verify" tooltip looks like it should (like on master) i mean colors, paddings etc.